src/disease.c

```diff
@@ -275,7 +277,11 @@ short seed_infect_by_idx(
 	if( infected->status != SUSCEPTIBLE )
 		return FALSE;
 
-	infected->infection_events->strain_multiplier = strain_multiplier;
+	if( model->strains[ strain_idx ].idx == -1 ) // if strain is not initialised
```

Comment:
We shouldn't have this function adding/updating strains as well as infecting people.

We should have a new function add_new_variant( model, transmission_multiplier), which returns the index of the new initialised strain. Then seed_infect_by_idx should only take strain_idx for pre-exisiting strains.